From 0f22bd18d5bf0e28641732a4d965ce9585727345 Mon Sep 17 00:00:00 2001
From: Pavan Kumar S <pavan.kumar.s@intel.com>
Date: Mon, 4 Nov 2013 08:37:52 +0530
Subject: [PATCH 111/429] EM/ADC: Append an empty entry to the iio_map list

iio_map_array_register API expects an empty entry in the iio_map table registered.

An empty entry should be appended in the GPADC iio-table.

Category: device enablement
Domain: EM.BatteryManagement-Common
Origin: internal
Upstream-Candidate: no, not-aosp

Signed-off-by: Pavan Kumar S <pavan.kumar.s@intel.com>
---
 arch/x86/platform/intel-mid/device_libs/platform_bcove_adc.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/x86/platform/intel-mid/device_libs/platform_bcove_adc.c b/arch/x86/platform/intel-mid/device_libs/platform_bcove_adc.c
index 44c898a4f663..5ac2bc2ddf69 100644
--- a/arch/x86/platform/intel-mid/device_libs/platform_bcove_adc.c
+++ b/arch/x86/platform/intel-mid/device_libs/platform_bcove_adc.c
@@ -45,6 +45,7 @@ struct iio_map iio_maps[] = {
 	MSIC_ADC_MAP("CH7", "bcove_thrm", "SYSTEMP1"),
 	MSIC_ADC_MAP("CH8", "bcove_thrm", "SYSTEMP2"),
 	MSIC_ADC_MAP("CH3", "bcove_thrm", "PMICTEMP"),
+	{ },
 };
 
 void __init *bcove_adc_platform_data(void *info)
-- 
2.37.3

